epifor/data/csse.py

```diff
@@ -23,27 +24,61 @@
 class CSSEData:
     def __init__(self):
         self.df = None
-
-    def load(self, pattern):
+        self.HIST_FILE_NAME = "csse_history_data.h5"
+
+    @staticmethod
+    def convert_date(string):
+        " Convert string from csv to pandas allowed format"
+        try:
+            return datetime.datetime.strptime(string, "%m/%d/%Y").strftime("%Y%m%d")
+        except ValueError:
+            return datetime.datetime.strptime(string, "%m/%d/%y").strftime("%Y%m%d")
+
+    @staticmethod
+    def nearest_date(items, pivot):
+        " Find the nearest date to start_date from config "
```

Comment:
Docstrings  should use `"""`, I know it's wrong in the rest of the module but I would say it's still better to stick to a standard for a new code.
```suggestion
        """Find the nearest date to start_date from config"""
```

Reply:
Corrected docstrings in files I have edited.